CMakeLists.txt

```diff
@@ -63,3 +63,17 @@ if (pfs_BUILD_TESTS)
     add_executable (unittest ${pfs_UNITTEST_SOURCES})
     target_link_libraries (unittest PRIVATE pfs)
 endif()
+
+include(CMakePackageConfigHelpers)
+configure_package_config_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/cmake/${CMAKE_PROJECT_NAME}-config.cmake.in
```

Comment:
There is some repetition with the values which is not obvious without deep inspection and makes it harder to understand. Would you mind extracting those into helper variables?
```
set (pfs_PACKAGE_CONFIG ${CMAKE_PROJECT_NAME}-config.cmake)
set (pfs_PACKAGE_CONFIG_IN ${CMAKE_CURRENT_SOURCE_DIR}/cmake/${pfs_PACKAGE_CONFIG}.in)
set (pfs_PACKAGE_CONFIG_OUT ...)

set (pfs_PACKAGE_INSTALL_DESTINATION ...)
```